docs/source/intro/run_docker.rst

```diff
@@ -0,0 +1,61 @@
+.. _intro:get_started:docker:
+.. _intro:install:docker:
+
+****************************
+Run AiiDA via a Docker image
```

Comment:
This removes a number of helpful commands from the original "Using the Docker image" section, for example the one you need to actual enter the container:

```
docker exec -it --user aiida aiida-container /bin/bash
```

plus the note about using ``/tmp/my_data``, which is the only place writable by the aiida user, and the command to mount it

Reply:
Yeah, I was actually a bit confused about that section. I'll restore the things that got removed, but I also want to avoid making this too much of a general docker tutorial. But I'm slightly less concerned about the length now that the instructions are on their own separate page.